### PR TITLE
Adjusted combined given/when steps in OccContext

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Combined `given-when` steps splitted into seperate functions and check-success assertion added on given steps.

## Related Issue
- Part of https://github.com/owncloud/QA/issues/635

## How Has This Been Tested?

- CI
- :robot:

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
closed to fix test failing issues!

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -412,121 +762,186 @@ public function theAdministratorSetsLogRotateFileSizeUsingTheOccCommand($size) {
...
 	 * @Given the administrator has changed the background jobs mode to :mode
 	 *
 	 * @param string $mode
 	 *
 	 * @return void
+	 * @throws Exception
 	 */
 	public function theAdministratorHasChangedTheBackgroundJobsModeTo($mode) {
-		$this->invokingTheCommand("background:$mode");
+		$this->changeBackgroundJobsModeToAsAdmin($mode);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: check the result?

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 2]
```diff
@@ -412,121 +762,186 @@ public function theAdministratorSetsLogRotateFileSizeUsingTheOccCommand($size) {
...
 	 * @param string $user
 	 *
 	 * @return void
+	 * @throws Exception
 	 */
 	public function theAdministratorScansTheFilesystemForUserUsingTheOccCommand($user) {
-		$this->invokingTheCommand(
-			"files:scan $user"
-		);
+		$this->scanFileSystemForAUserAsAdminAsAdmin($user);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: remove duplication in the function name

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 3]
```diff
@@ -536,29 +951,35 @@ public function theAdministratorCleanupsTheFilesystemForAllUsersUsingTheOccComma
...
-	 * @param string $userOrGroup
-	 * @param string $user
+	 * @param string $mount
 	 *
 	 * @return void
-	 * @throws \Exception
 	 */
-	public function theAdminAddsRemovesAsTheApplicableUserLastLocalMountUsingTheOccCommand(
+	public function theAdministratorHasCreatedTheLocalStorageMountForAUserUsingTheOccCommand($mount) {
+		$this->asAdminCreateLocalStorageMountForAUser($mount);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: check result?

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Let's merge this, and then adjust a few method names etc in a next PR.

**Comment by phil-davis** [comment]
PR #36682 cleans up

**Pull request merged**